dockerfiles/Dockerfile.nemo-rl

```diff
@@ -0,0 +1,62 @@
+# copied and edited from https://github.com/NVIDIA/NeMo-RL/blob/main/docker/Dockerfile
+ARG BASE_IMAGE=nvcr.io/nvidia/cuda:12.8.1-cudnn-devel-ubuntu24.04
+FROM ${BASE_IMAGE} AS base
+
+# It is more convenient for users to run as root
+USER root
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    jq \
+    curl \
+    git \
+    && rm -rf /var/lib/apt/lists/* && \
+    apt-get clean
+
+# Install uv and python
+ARG UV_VERSION=0.7.2
+ARG PYTHON_VERSION=3.12
+ENV PATH="/root/.local/bin:$PATH"
+RUN curl -LsSf https://astral.sh/uv/${UV_VERSION}/install.sh | sh && \
+    uv python install ${PYTHON_VERSION}
+
+# Disable usage stats by default for users who are sensitive to sharing usage.
+# Users are encouraged to enable if the wish.
+ENV RAY_USAGE_STATS_ENABLED=0
+
+FROM base AS hermetic
+
+ARG NEMO_RL_COMMIT
+ENV NEMO_RL_COMMIT=${NEMO_RL_COMMIT:-cec9a60ff798554279ca494a0ef40ce5f283e0d8}
```

Comment:
Should we use this commit then 8e155f9427d9403086505f55c106339957fcba04? We can update dockerfile after that PR is merged, but this will make sure it works out-of-the-box without a custom mount, right?

Reply:
Yeah that's right

Reply:
So do you want to change the default commit here?

Reply:
yes, done